src/gps/GPS.cpp

```diff
@@ -1138,35 +1143,10 @@ GnssModel_t GPS::probe(int serialSpeed)
     // Check that the returned response class and message ID are correct
     GPS_RESPONSE response = getACK(0x06, 0x08, 750);
     if (response == GNSS_RESPONSE_NONE) {
-        LOG_WARN("Failed to find UBlox & MTK GNSS Module using baudrate %d\n", serialSpeed);
+        LOG_WARN("Failed to find GNSS Module (baudrate %d)\n", serialSpeed);
         return GNSS_MODEL_UNKNOWN;
     } else if (response == GNSS_RESPONSE_FRAME_ERRORS) {
-        LOG_INFO("UBlox Frame Errors using baudrate %d\n", serialSpeed);
-    } else if (response == GNSS_RESPONSE_OK) {
-        LOG_INFO("Found a UBlox Module using baudrate %d\n", serialSpeed);
-    }
-
-    // tips: NMEA Only should not be set here, otherwise initializing Ublox gnss module again after
-    // setting will not output command messages in UART1, resulting in unrecognized module information
-    if (serialSpeed != 9600) {
-        // Set the UART port to 9600
-        uint8_t _message_prt[] = {0xB5, 0x62, 0x06, 0x00, 0x14, 0x00, 0x01, 0x00, 0x00, 0x00, 0xD0, 0x08, 0x00, 0x00,
-                                  0x80, 0x25, 0x00, 0x00, 0x07, 0x00, 0x03, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00};
-        UBXChecksum(_message_prt, sizeof(_message_prt));
-        _serial_gps->write(_message_prt, sizeof(_message_prt));
-        delay(500);
-        serialSpeed = 9600;
-#if defined(ARCH_NRF52) || defined(ARCH_PORTDUINO) || defined(ARCH_STM32WL)
-        _serial_gps->end();
-        _serial_gps->begin(serialSpeed);
-#elif defined(ARCH_RP2040)
-        _serial_gps->end();
-        _serial_gps->setFIFOSize(256);
-        _serial_gps->begin(serialSpeed);
```

Comment:
I assume Fifo Size is already 256 here, just reopening?

Reply:
I believe so - checking.

Reply:
Yes , set at the start of  GPS::probe

Reply:
(line 1093 or so)